### DISCUSSION TITLE
SuiteAdvanced for power users

### DISCUSSION THREAD

**Discussion by InitiativeWorking** [burkybang/SuiteAdvanced-Feedback#111] [Ideas]
I had a colleague ask about my NetSuite since it was very different from theirs.
I showed them SuiteAdvanced and they might be interested in using it, but as they are not NetSuite admins, they don't need most of the functionalities in the standard SuiteAdvanced version.

I was wondering if there is a plan to have SuiteAdvanced versions at different prices or maybe roles that could be customized by an organization admin, for example. I didn't seem to find anything about it so here's a "formal" idea post.

The idea would be allowing specific power users to have access to certain SuiteAdvanced functionalities, like the custom display, the field help, the collapsible fields, the dashboard functionalities, etc.

Hope this might be interesting/helpful for you 🙂

**Comment by burkybang** [comment]
Thanks. Will consider this

**Comment by HunterJacobs** [comment]
Honestly something that just has the refresh functionality and maybe themes would be an amazing feature! I've seen many departments constantly refresh the page to update reports/work.
